pt-BR/form_helpers.md

```diff
@@ -991,7 +991,7 @@ As a convenience you can instead pass the symbol `:all_blank` which will create
 
 Rather than rendering multiple sets of fields ahead of time you may wish to add them only when a user clicks on an 'Add new address' button. Rails does not provide any built-in support for this. When generating new sets of fields you must ensure the key of the associated array is unique - the current JavaScript date (milliseconds since the [epoch](https://en.wikipedia.org/wiki/Unix_time)) is a common choice.
 
-Using form_for and form_tag
+Usando _form_for_ e _form_tag_
 ---------------------------
 
-Before `form_with` was introduced in Rails 5.1 its functionality used to be split between `form_tag` and `form_for`. Both are now soft-deprecated. Documentation on their usage can be found in [older versions of this guide](https://guides.rubyonrails.org/v5.2/form_helpers.html).
\ No newline at end of file
+Antes do `form_with` ser introduzido no _Rails_ 5.1 sua funcionalidade costumava ser divida entre `form_tag` e `form_for`. Ambos estão agora depreciados (_soft-deprecated_). A documentação sobre seu uso pode ser encontrada na [versão antiga deste guia](https://guides.rubyonrails.org/v5.2/form_helpers.html).
```

Comment:
```suggestion
Antes do `form_with` ser introduzido no Rails 5.1 sua funcionalidade costumava ser divida entre `form_tag` e `form_for`. Ambos estão agora depreciados (_soft-deprecated_). A documentação sobre seu uso pode ser encontrada na [versão antiga deste guia](https://guides.rubyonrails.org/v5.2/form_helpers.html).
```

Como Rails é um nome próprio não acho que deveria ser itálico como estrangeirismo. Essa é um pouco confusa, talvez no manual devesse ser mais direto. O que acha?

Reply:
Legal, acho que  faz sentido. Coloquei o _itálico_ pela regra mesmo, talvez seja legal adicionar isso como exceção lá ;)